packages/features/steps/app.steps.ts

```diff
@@ -1,13 +1,26 @@
-import {executables} from '../lib/constants'
-import {exec} from '../lib/system'
+import {executables} from '../lib/constants.js'
+import {exec} from '../lib/system.js'
 import * as path from 'pathe'
 import {When, Then} from '@cucumber/cucumber'
 import fs from 'fs-extra'
 import {strict as assert} from 'assert'
 
+interface AppInfo {
+  extensions: {
+    [category: string]: Extension[]
+  }
+  allExtensions: Extension[]
```

Comment:
does this work? we removed `extensions.{category}` in favor of `allExtensions` in the `AppInterface` 🤔